src/Surfnet/Stepup/Configuration/Event/InstitutionConfigurationRemovedEvent.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\Stepup\Configuration\Event;
+
+use Surfnet\Stepup\Configuration\Value\Institution;
+
+class InstitutionConfigurationRemovedEvent
```

Comment:
This event could use some serialize/deserialize methods. Please add them (mind the tests).